targets/csharp/make.js

```diff
@@ -56,7 +54,9 @@ function getBaseTypeSyntax(datatype) {
 
     if (datatype.className.toLowerCase().endsWith("request"))
         parents.push("PlayFabRequestCommon");
-    if (datatype.className.toLowerCase().endsWith("response") || datatype.className.toLowerCase().endsWith("result"))
+    if (datatype.className.toLowerCase() === "loginresult")
+        parents.push("PlayFabLoginResultCommon");
+    else if (datatype.className.toLowerCase().endsWith("response") || datatype.className.toLowerCase().endsWith("result"))
         parents.push("PlayFabResultCommon");
     if (datatype.sortKey)
```

Comment:
Let's introduce a blankline here, and a new comment, separating it as the begining of the interface parents, of which there can be as many as is deemed necessary.
Basically, I want to reinforce the inheritance rules of C# here, so that we don't accidentally generate invalid code.